### PR TITLE
DOCSP-35840 Mention OnlineArchive on compass MaxTimeMS docs

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## DESCRIPTION
Mentions OnlineArchive on compass MaxTimeMS page.

## STAGING
https://preview-mongodbkanchanamongodb.gatsbyjs.io/compass/DOCSP-35840/query/maxtimems/

## JIRA
https://jira.mongodb.org/browse/DOCSP-35840

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65ccf00ecbcd269fa52d56d7 (clean)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/query/maxtimems.txt [hunk 1]
```diff
@@ -14,9 +14,11 @@ interrupts the operation.
    :alt: MaxTimeMS Option
 
 The default :guilabel:`MAX TIME MS` value is 60000, or 60 seconds.
-Consider raising this value if you have a very large collection, or
-your operations are frequently timing out. You can also consider creating
-:ref:`indexes <compass-indexes>` to improve query performance.
+Consider raising this value if you have a very large collection, if 
+your operations are frequently timing out, or if you are querying
+data archived from the Atlas cluster using Online Archive. You can also
```
Comment (ltran-mdb2) [inline source/query/maxtimems.txt:19]: I might suggest breaking this sentence up into bullets for improved readability 

```suggestion
Consider raising this value if you meet one of the following conditions:

- You have a very large collection.
- Your operations frequently time out.
- You query data archived from the Atlas cluster using Online Archive. 

You can also
```

File: source/query/maxtimems.txt [hunk 2]
```diff
@@ -14,9 +14,11 @@ interrupts the operation.
    :alt: MaxTimeMS Option
 
 The default :guilabel:`MAX TIME MS` value is 60000, or 60 seconds.
-Consider raising this value if you have a very large collection, or
-your operations are frequently timing out. You can also consider creating
-:ref:`indexes <compass-indexes>` to improve query performance.
+Consider raising this value if you have a very large collection, if
```
Comment (ltran-mdb2) [inline source/query/maxtimems.txt:17]: [question] is there a way to more concretely quantify a "very large" collection?

**Review by ltran-mdb2 — COMMENTED** [review]
Thank for picking up this work @kanchana-mongodb ! I left a suggestion and question for your consideration.

File: source/query/maxtimems.txt (hunk 2 above)
Comment (kanchana-mongodb) [reply source/query/maxtimems.txt]: There is no limit on the number of documents in a non-capped collection, but I think this depends on the hardware.

**Review by xstephen95x — APPROVED** [review]
lgtm! thank you for calling out Online Archive here.

**Review by ltran-mdb2 — APPROVED** [review]
LGTM! Thanks @kanchana-mongodb !

**Pull request merged**